### ISSUE TITLE
gfx_device_gl 0.16.1 compilation error on latest nightly - "could not find `format_args` in `core`"

### ISSUE THREAD

**Issue by mitchmindtree** [gfx-rs/gfx#2964]
Currently attempting to update conrod, however [travis's nightly build seems to be failing](https://travis-ci.org/PistonDevelopers/conrod/jobs/573432734#L1120-L1147) with the following error. Curiously, stable and beta build fine. I wonder if this is a bug in the latest rustc nightly?

```
   Compiling gfx_device_gl v0.16.1

     Running `rustc --crate-name gfx_device_gl /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/gfx_device_gl-0.16.1/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C debuginfo=2 -C metadata=99ad1df062eeb806 -C extra-filename=-99ad1df062eeb806 --out-dir /home/travis/build/PistonDevelopers/conrod/target/debug/deps -L dependency=/home/travis/build/PistonDevelopers/conrod/target/debug/deps --extern gfx_core=/home/travis/build/PistonDevelopers/conrod/target/debug/deps/libgfx_core-1411fbda9e20854f.rmeta --extern gfx_gl=/home/travis/build/PistonDevelopers/conrod/target/debug/deps/libgfx_gl-ecbe19e745d0b4a3.rmeta --extern log=/home/travis/build/PistonDevelopers/conrod/target/debug/deps/liblog-7a40360aa6455ce2.rmeta --cap-lints allow`

error[E0433]: failed to resolve: could not find `format_args` in `core`

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/gfx_device_gl-0.16.1/src/shade.rs:434:24

    |

434 |                 name = format!("Target{}", index);

    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `format_args` in `core`

    |

    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error[E0433]: failed to resolve: could not find `format_args` in `core`

   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/gfx_device_gl-0.16.1/src/shade.rs:487:30

    |

487 |             let color_name = format!("Target{}\0", i);

    |                              ^^^^^^^^^^^^^^^^^^^^^^^^ could not find `format_args` in `core`

    |

    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0433`.

error: Could not compile `gfx_device_gl`.

Caused by:

  process didn't exit successfully: `rustc --crate-name gfx_device_gl /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/gfx_device_gl-0.16.1/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C debuginfo=2 -C metadata=99ad1df062eeb806 -C extra-filename=-99ad1df062eeb806 --out-dir /home/travis/build/PistonDevelopers/conrod/target/debug/deps -L dependency=/home/travis/build/PistonDevelopers/conrod/target/debug/deps --extern gfx_core=/home/travis/build/PistonDevelopers/conrod/target/debug/deps/libgfx_core-1411fbda9e20854f.rmeta --extern gfx_gl=/home/travis/build/PistonDevelopers/conrod/target/debug/deps/libgfx_gl-ecbe19e745d0b4a3.rmeta --extern log=/home/travis/build/PistonDevelopers/conrod/target/debug/deps/liblog-7a40360aa6455ce2.rmeta --cap-lints allow` (exit code: 1)

warning: build failed, waiting for other jobs to finish...

error: build failed

The command "cargo test --verbose" exited with 101.
```

I believe this is related to the pre-ll gfx.

**Comment by TheOpenDevProject** [comment]
This is also blocking me from building my project.

**Comment by alexheretic** [comment]
I noticed this recently as I wanted to mess about with some profile-overrides (nightly only). Refactoring from `extern crate gfx_core as core` -> `extern crate gfx_core` seems to fix currently nightly compilation.

I'd say this is a rustc regression, but it's pretty harmless to fix the pre-ll code too. Particularly as the `core` alias seems a poor one anyway. Would you guys like a pr?

**Comment by alexheretic** [comment]
This is fixed with `0.16.2`

**Comment by GabrielMajeri** [comment]
@kvark this issue seems to have been fixed
